### PR TITLE
feat: templates for week pages

### PR DESCRIPTION
**Pull request by stdword**
## Insert template for empty weekly pages

### CODE REVIEW

File: src/comps/Calendar.jsx [hunk 1]
```diff
@@ -75,34 +77,67 @@ export default function Calendar({
 
   async function gotoJournal(d, openInSidebar) {
     const pageName = format(d, dateFormat, { locale: enUS })
-    const dayData = days.get(d.getTime())
-    if (dayData?.uuid) {
-      if (openInSidebar) {
-        const page = await logseq.Editor.getPage(pageName)
-        logseq.Editor.openInRightSidebar(page.uuid)
```
Comment (stdword) [inline src/comps/Calendar.jsx:82]: If page doesn't exist, here was a console error for page.uuid

Comment (sethyuan) [reply src/comps/Calendar.jsx:82]: It's not code duplication actually, they're slightly different. 😂

**Comment by sethyuan** [comment]
BTW, can you tune your IDE to use prettier? It takes care of all code formatting inconsistencies.

File: src/comps/Calendar.jsx (hunk 1 above)
Comment (stdword) [reply src/comps/Calendar.jsx:82]: I've mean that part (screenshot of Before):

**Comment by stdword** [comment]
> BTW, can you tune your IDE to use prettier? It takes care of all code formatting inconsistencies.

Sure! Just committed the changes. I was confused as prettier is not in package.json and build scripts.

**Comment by sethyuan** [comment]
I released a new version, FYI.

**Pull request merged**